pkg/network/common/list_pager.go

```diff
@@ -0,0 +1,110 @@
+package common
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/pager"
+
+	osdnv1 "github.com/openshift/api/network/v1"
+	osdnclient "github.com/openshift/client-go/network/clientset/versioned"
+)
+
+func ListAllEgressNetworkPoliciesWithPager(client osdnclient.Interface) ([]*osdnv1.EgressNetworkPolicy, error) {
+	list := []*osdnv1.EgressNetworkPolicy{}
+	err := pager.New(func(ctx context.Context, opts metav1.ListOptions) (runtime.Object, error) {
+		return client.NetworkV1().EgressNetworkPolicies(metav1.NamespaceAll).List(ctx, opts)
+	}).EachListItem(context.TODO(), metav1.ListOptions{}, func(obj runtime.Object) error {
```

Comment:
This (and all similar methods) should take a Context as their first argument, rather than doing TODO here.